### ISSUE TITLE
Unable to create a centaur

### ISSUE THREAD

**Issue by RyanGetter** [lllyasviel/Fooocus#2282]
**Read Troubleshoot**

[x] I confirm that I have read the [Troubleshoot](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md) guide before making this issue.

**I was inspired by something I saw on YouTube to make a Donald Trump Centaur, but slowly realized I just couldn't make one at all. I'm brand spanking new to this, so maybe there just is something I'm doing wrong, but it just kept making him ride a horse, sometimes shirtless. I tried using the negative prompts to make it not show horses, but it just kept showing horses, in fact this time there was no rider, just more horses, so somehow it made it worse. Anyways, I'm sorry i didn't include the code or even recreated the issue, because I was, like, banned or something. I'm guessing for having it draw children. For Valentine's Day, I asked my mother what would she like to experience more than anything again, and she said picking up seashells on the Savannah, Georgia beach in the late 60's , and it did surprisingly well at this. I was going to work on the images more, but got deemed ''not a priority'' or something like that, and I get it. I probably wasn't contributing much and I hate to waste your guys' GPU's. I mean i don't even know how to write the most basics of code. I made this account solely because some guy said on Youtube said that you guys were the best.**

**Unable to do this now, and I couldn't even recreate it if I wanted to because i'm not a priority.**
Paste the **full** console log here. You will make our job easier if you give a **full** log.

**Comment by mashb1t** [comment]
1. You can deactivate all styles to first figure out the prompt as well as then iteratively add more styles if you got it working.
2. Please check which resolution you're using.Ensure that your Canvas reflects maybe a wider aspect ratio like 9:7 depending how you want to depict the centaur.
3. Fooocus uses the general purpose model JuggernautXL v8, which should be able to create centaurs. If this really doesn't work you can use a different model by downloading one from e.g. civitai.com

What i'd do is to create the image of a centaur and then inpaint a specific face/upper body onto it.

**Comment by RyanGetter** [comment]
tried again with nothing selected

it made naked ladies with no additional prompts. i only used the word centaur. 

[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 8899922347845010457
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] centaur, adapted, highly detailed, sharp focus, cinematic composition, warm colors, modern, exciting, advanced, beautiful, attractive, intricate, very stylish, dramatic ambient background, thought, fancy, full color, epic light, atmosphere, best, spectacular, fresh, creative, positive, unique, appealing, perfect, trendy, futuristic, artistic, vibrant, scientific, surreal
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] centaur, intricate, elegant, highly detailed, strong background, cinematic, lush, dramatic light, sharp focus, inspired bright colors, open dynamic color, professional composition, expressive, fair, true artistic, emotional, shiny, beautiful, epic, striking, attractive, symmetry, great quality, very inspirational, clear, winning, best, creative, positive, cute, lovely
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.12 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 6.10 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.65 seconds
100% 30/30 [00:26<00:00,  1.13it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.33 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 32.15 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.81 seconds
100% 30/30 [00:24<00:00,  1.23it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.40 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 30.05 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.86 seconds
Total time: 72.84 seconds

**Comment by RyanGetter** [comment]
clicked advanced and removed all styles. nothing

then tried each of the four fantasy models one by one and had set to create two images,  only once did it almost make a centaur, one of the rides legs were morphed into the horse. here's the code.

i'm not at any way complaining; i'm trying to find my own way to contribute.

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from Terminal to deploy to Spaces (https://huggingface.co/spaces)
2024-02-18 23:00:47.374804: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-02-18 23:00:47.374850: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-02-18 23:00:47.376378: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-02-18 23:00:48.677240: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection'}
Base model loaded: /content/Fooocus/models/checkpoints/juggernautXL_v8Rundiffusion.safetensors
Request to load LoRAs [['sd_xl_offset_example-lora_1.0.safetensors', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [/content/Fooocus/models/checkpoints/juggernautXL_v8Rundiffusion.safetensors].
Loaded LoRA [/content/Fooocus/models/loras/sd_xl_offset_example-lora_1.0.safetensors] for UNet [/content/Fooocus/models/checkpoints/juggernautXL_v8Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cuda:0, use_fp16 = True.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.83 seconds
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865 or https://4bb5f48c36b0519e54.gradio.live
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 8899922347845010457
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] centaur, adapted, highly detailed, sharp focus, cinematic composition, warm colors, modern, exciting, advanced, beautiful, attractive, intricate, very stylish, dramatic ambient background, thought, fancy, full color, epic light, atmosphere, best, spectacular, fresh, creative, positive, unique, appealing, perfect, trendy, futuristic, artistic, vibrant, scientific, surreal
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] centaur, intricate, elegant, highly detailed, strong background, cinematic, lush, dramatic light, sharp focus, inspired bright colors, open dynamic color, professional composition, expressive, fair, true artistic, emotional, shiny, beautiful, epic, striking, attractive, symmetry, great quality, very inspirational, clear, winning, best, creative, positive, cute, lovely
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.12 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 6.10 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.65 seconds
100% 30/30 [00:26<00:00,  1.13it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.33 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 32.15 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.81 seconds
100% 30/30 [00:24<00:00,  1.23it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.40 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 30.05 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.86 seconds
Total time: 72.84 seconds
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 4785229233624284894
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 60 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.12 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (1152, 896)
Preparation time: 0.37 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.42 seconds
100% 60/60 [00:46<00:00,  1.28it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.31 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 52.19 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.91 seconds
100% 60/60 [00:48<00:00,  1.24it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.33 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 53.22 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.87 seconds
Total time: 110.84 seconds
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 735965036278290600
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 60 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.13 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (1152, 896)
Preparation time: 0.37 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.42 seconds
100% 60/60 [00:48<00:00,  1.24it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.31 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 53.64 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.89 seconds
100% 60/60 [00:49<00:00,  1.21it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.31 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 54.43 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 1.30 seconds
Total time: 113.27 seconds
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 221037834699020455
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 60 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.14 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (1152, 896)
Preparation time: 0.43 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.69 seconds
100% 60/60 [00:48<00:00,  1.24it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.37 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 54.13 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.95 seconds
100% 60/60 [00:49<00:00,  1.21it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.31 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 54.35 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.89 seconds
Total time: 113.87 seconds
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 417736119977674490
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 60 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.12 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (1152, 896)
Preparation time: 0.36 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.45 seconds
100% 60/60 [00:47<00:00,  1.25it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.31 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 53.30 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.35 seconds
100% 60/60 [00:49<00:00,  1.22it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.34 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 54.51 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 0.88 seconds
Total time: 112.83 seconds
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 1102315401643777176
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 60 - 30
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.13 seconds
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (1152, 896)
Preparation time: 0.37 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.45 seconds
100% 60/60 [00:48<00:00,  1.24it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.30 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 53.71 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 1.92 seconds
100% 60/60 [00:49<00:00,  1.21it/s]
Requested to load AutoencoderKL
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.30 seconds
Image generated with private log at: /content/Fooocus/outputs/2024-02-18/log.html
Generating and saving time: 54.25 seconds
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
[Fooocus Model Management] Moving model(s) has taken 1.03 seconds
Total time: 113.10 seconds

**Comment by mashb1t** [comment]
@RyanGetter the last 30min i've also tried my best to get centaurs to work

I found this LoRA: https://civitai.com/models/139065/rpgcentaurxl
Please check it out!

![image - 2024-02-19T004702 688](https://github.com/lllyasviel/Fooocus/assets/9307310/992e93c0-cc5d-4c5c-bf30-e50d791b3f9e)
![image - 2024-02-19T004701 006](https://github.com/lllyasviel/Fooocus/assets/9307310/018a6e6e-283a-49f3-90b8-00e2f8dd643b)


Without it so far no luck.
Sadly Fooocus doesn't support the attention shift for prompt blending like A1111 does (https://www.andyhtu.com/post/prompt-weights-punctuations-how-to-use-it-in-automatic1111-stable-diffusion#viewer-679q9).
Also, "horse" seems very dominant in almost every prompt, even with a low weight of 0.2.
I was able to generate pigs with alligators, but no centaurs.

![image - 2024-02-19T002948 085](https://github.com/lllyasviel/Fooocus/assets/9307310/0edb23c9-a7dd-42f6-aadd-2eba03ab76ae)


![image - 2024-02-19T003105 876](https://github.com/lllyasviel/Fooocus/assets/9307310/9f8619c2-7771-479f-9e49-51c08c30fbc0)

Please check the LoRA!

**Comment by RyanGetter** [comment]
didn't realize you had responded, i'm sorry
